indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -49,21 +60,46 @@ export default function EditableList() {
       codePresent={codePresent}
       editableType={type}
       eventId={eventId}
+      editors={editors}
     />
   );
 }
 
-function EditableListDisplay({editableList, codePresent, editableType, eventId}) {
+function EditableListDisplay({
+  editableList: origEditableList,
+  codePresent,
+  editableType,
+  eventId,
+  editors,
+}) {
   const [sortBy, setSortBy] = useState('friendly_id');
   const [sortDirection, setSortDirection] = useState('ASC');
+  const [editableList, setEditableList] = useState(origEditableList);
   const [sortedList, setSortedList] = useState(editableList);
   const [checked, setChecked] = useState([]);
-  const hasCheckedRows = checked.length === 0;
+  const editables = editableList.filter(x => x.editable);
+  const hasCheckedEditables = checked.length > 0;
+  const checkedSet = new Set(checked);
+  const checkedEditables = editables.filter(x => checkedSet.has(x.editable.id));
+  const [activeRequest, setActiveRequest] = useState(null);
+
+  const editorOptions = useMemo(
+    () =>
+      _.sortBy(editors, 'fullName').map(e => ({
```

Comment:
Do we need to case-insensitive sorting here as we have done in case of titles?

Reply:
+-0 on that, usually the first letter of a name is an uppercase latin character or a character where it doesn't really matter whether or not to do it case-insensitively